modules/uhttp/http.go

```diff
@@ -154,7 +154,7 @@ func (m *Module) Start(ready chan<- struct{}) <-chan error {
 		router.Handle(h.Path, m.fcb.Build(h.Handler))
 	}
 
-	if m.config.Debug == nil || *m.config.Debug {
+	if m.config.Debug {
```

Comment:
I prefer to keep pprof true by default, unless users really know what they are doing.